Add changelog item for proposal comment expansion (#293)
apollographql/apollo-studio-community: CHANGELOG.md

[comment]: <> "NOTE! Ensure all images are added via the \[label\]\(link\) syntax!"

## 2024-12-09

| What's new | |
| :--------- | :-: |

> | We've made it possible to comment on lines containing field return types and descriptions in Schema Proposals. Clicking on the comment button on the Proposal Changes page on these lines now lets you target more specific parts of your schema more easily! | ![image](https://github.com/user-attachments/assets/ac75f026-8891-4b30-a099-ab8682482533) 

Reviewer: Couple of suggestions for consistency and conciseness.
(We haven't generally capitalized "schema proposals" in the docs and try to stay away from using "!" too frequently.)

```suggestion
| You can now comment on lines containing field return types and descriptions in schema proposals. This lets you target more specific parts of your schema more easily. | ![image](https://github.com/user-attachments/assets/ac75f026-8891-4b30-a099-ab8682482533) 
```
